features/gardener/pkg.exclude

```diff
@@ -0,0 +1,2 @@
+selinux-basics
+selinux-policy-default
```

Comment:
Why do you want to exclude those packages?

Reply:
They should not be removed, this was a misunderstanding.